src/libaktualizr-c/test/Makefile

```diff
@@ -0,0 +1,11 @@
+CC=gcc
+CFLAGS=-I/home/esm/projects/aktualizr/include -g -O0
+# Run application with LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/esm/projects/aktualizr/build/src/libaktualizr-c
+LIBS=-L/home/esm/projects/aktualizr/build/src/libaktualizr-c -laktualizr-c
+OBJ=api-test.o
+
+%.o: %.c
+	$(CC) -c -o $@ $< $(CFLAGS)
+
+api-test: $(OBJ)
+	$(CC) -o $@ $^ $(CFLAGS) $(LIBS)
```

Comment:
Why not use cmake? And either way, is there a way to integrate the test into the ctest framework?

Reply:
I started with having this test app completely separate, to be sure that linking works without any implicit cmake magic. But I guess I'll use cmake and add it to the ctest now.